language/stdlib/modules/Association.move

```diff
@@ -36,6 +36,11 @@ module Association {
     /// means that the address is an association address.
     struct Association { }
 
+    /// The presence of an instance of an `Association::ModulePublishing`
+    /// privilege at an address means that that address can publish code to
+    /// the chain.
+    struct ModulePublishing { }
```

Comment:
Nit: I am trying to use present tense, verb first (e.g. `PublishModule`) names for all capability/script names (since that seems to be the convention we follow most consistently, though not perfectly).